runner/runner.go

```diff
@@ -83,6 +88,8 @@ func init() {
 	flag.StringVar(&logDest, "log", "os.Stdout", "Log path, omit to log to Stdout")
 	flag.StringVar(&logFormat, "log-format", "JSON", "Log format, 'Text' or 'JSON'")
 	flag.BoolVar(&debug, "debug", false, "Include additional log data for debugging")
+	flag.BoolVar(&fpm, "fpm", false, "Use FPM to run jobs")
+	flag.StringVar(&fpmUrl, "fpm-url", "unix:///var/run/fastcgi.sock", "Url for the php-fpm server or socket")
```

Comment:
I would just default this to the empty string `""` and providing a value here also enables fpm, so we do not need a separate book variable. This is fairly standard golang, using the "zero" value of a type to represent absence.

Reply:
Ok. will do